src/applications/coronavirus-chatbot/createCoronavirusChatbot.js

```diff
@@ -7,8 +10,23 @@ export default (_store, widgetType) => {
     return;
   }
 
-  import(/* webpackChunkName: "chatbot" */ './index').then(module => {
-    const initializeChatbot = module.default;
-    initializeChatbot(root);
-  });
+  import(/* webpackChunkName: "chatbot" */ './index')
+    .then(module => {
+      const initializeChatbot = module.default;
+      initializeChatbot(root);
+    })
+    // eslint-disable-next-line no-unused-vars
+    .then(res => {
+      recordEvent({
+        event: `${GA_PREFIX}-load-successful`,
+        'error-key': undefined,
+      });
+    })
```

Comment:
If the promise callback isn't using the argument, would it be possible to just move the `recordEvent` call up to right after `initializeChatbot` is called?  That way ESLint wouldn't have to be disabled, and it might make more sense.